UI/window-basic-main-outputs.cpp

```diff
@@ -420,6 +430,13 @@ SimpleOutput::SimpleOutput(OBSBasic *main_) : BasicOutputHandler(main_)
 							       : "ffmpeg_nvenc";
 		LoadStreamingPreset_Lossy(id);
 
+#ifdef ENABLE_HEVC
+	} else if (strcmp(encoder, SIMPLE_ENCODER_NVENC_HEVC) == 0) {
+		const char *id = EncoderAvailable("jim_hevc_nvenc")
+					 ? "jim_hevc_nvenc"
+					 : "ffmpeg_hevc_nvenc";
+		LoadStreamingPreset_Lossy(id);
+#endif
```

Comment:
Add a line break below to match the existing pattern.

Reply:
Done.